MATLAB/Algorithms/FDK.m

```diff
@@ -148,7 +152,7 @@
 % Ref: 
 %    Wang, Ge. X-ray micro-CT with a displaced detector array. Medical Physics, 2002,29(7):1634-1636.
 offset = geo.offDetector(1);
-offset = offset + (geo.DSD(1)/geo.DSO(1))*geo.COR(1);
+offset = offset + (geo.DSD(1) / geo.DSO(1)) * geo.COR(1);   % added correction

```

Comment:
We should remove `preweighting2()` totally and make `redundacy_weigting()` take the same arguments as `preweighting2()`. That way we don't have redundant code.